examples/api/lib/widgets/sliver/decorated_sliver.1.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+// Flutter code example for [DecoratedSliver]
+// with clipping turned off in a parent [CustomScrollView].
+
+void main() => runApp(const DecoratedSliverClipExampleApp());
+
+class DecoratedSliverClipExampleApp extends StatelessWidget {
+  const DecoratedSliverClipExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'DecoratedSliver Clip Example',
+      theme: ThemeData(
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+        useMaterial3: true,
+      ),
+      home: const DecoratedSliverClipExample(),
+    );
+  }
+}
+
+class DecoratedSliverClipExample extends StatefulWidget {
+  const DecoratedSliverClipExample({super.key});
+
+  @override
+  State<DecoratedSliverClipExample> createState() =>
+      _DecoratedSliverClipExampleState();
+}
+
+class _DecoratedSliverClipExampleState
+    extends State<DecoratedSliverClipExample> {
+  double _height = 225.0;
+  bool isClipped = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color(0xFF1C1C1C),
+      body: Column(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              Switch(
+                inactiveTrackColor: Colors.cyan,
+                activeColor: Colors.pink,
+                onChanged: (value) {
+                  setState(() {
+                    isClipped = value;
+                  });
+                },
+                value: isClipped,
+              ),
+              Slider(
+                activeColor: Colors.pink,
+                inactiveColor: Colors.cyan,
+                onChanged: (value) {
+                  setState(() {
+                    _height = value;
+                  });
+                },
+                value: _height,
+                min: 150,
+                max: 225,
+              ),
+            ],
+          ),
+          const SizedBox(
+            height: 20.0,
+          ),
+          Stack(
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
```

Comment:
```suggestion
                padding: const EdgeInsets.all(24.0),
```

Just to make it look nicer :)